lib/IdpListsServiceCsv.php

```diff
@@ -184,4 +194,30 @@ private function arrayToIdp($csv) {
 		return $idp;
 	}
 
+	/**
+	 * This function test the changes in file against the same file in repository.
+	 * @param $file string Path to file for git status
+	 * @return bool Return true, if file was change against repository, false if wasn't.
+	 */
+	private function gitStatus($file){
+		$idpListDir = dirname($file);
+		$fileName = basename($file);
+		exec("cd $idpListDir && git status $fileName", $output);
+		if ($output[1] === "nothing to commit, working directory clean"){
```

Comment:
You can use git status --porcelain, which prints nothing if there are no changes. If there is a change then it prints in parsable format.